src/NewProject/CMakeLists.txt

```diff
@@ -26,35 +26,17 @@ include_directories(${PROJECT_SOURCE_DIR}/../../src ${CMAKE_CURRENT_BINARY_DIR}/
 
 set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/../../lib)
 
+file(GLOB SRC_CPP_LIST "*.cpp" "ProjectManager/*.cpp")
```

Comment:
It is not such a good idea to GLOB files, it is better to have an explicit file list.

Reply:
I will change it back.
I don't have enough experience in this area, I just feel it looks very concise.